sway-lib-core/src/ops.sw

```diff
@@ -1215,6 +1215,122 @@ impl Shift for u16 {
     }
 }
 
+/// Trait to compare values of the same type.
+pub trait TotalOrd {
```

Comment:
I wonder if this should be in a new module, like `cmp.sw`. In rust, `Ord` and `PartialOrd` are in `std::cmp`. Should we do the same and put this in the std library instead of core?

Reply:
Never mind, just saw https://github.com/FuelLabs/sway/pull/6635#discussion_r1807539128